exporter/opensearchexporter/README.md

```diff
@@ -29,46 +29,46 @@ Index pattern will follow the next naming template `ss4o_{type}-{dataset}-{names
 - `dataset` (default=`default`) a user-provided label to classify source of telemetry. It is used to construct the name of the destination index or data stream.
 - `namespace` (default=`namespace`) a user-provided label to group telemetry. It is used to construct the name of the destination index or data stream.
 
-LogsIndex configures the index, index alias, or data stream name logs should be indexed in.
+#### Dynamic Indexing
 
-- `logs_index` a user-provided label to specify name of the destination index or data stream.
-
-#### Dynamic Log Indexing
-
-The OpenSearch exporter supports dynamic log index names using placeholders in the `logs_index` config. You can use any attribute or context key as a placeholder to construct index names dynamically per log record.
+The OpenSearch exporter supports dynamic index names for both logs and traces using placeholders. You can use any attribute or context key as a placeholder to construct index names dynamically per record.
 
 > Caution: In practice, OpenSearch clusters can become unstable or even break down when index counts reach a very high level. Introducing attributes with high cardinality resulting in many separate indices can significantly impact the stability of your target cluster.
 
-- Placeholder: `%{key}`
+**Configuration Options:**
 
-  - Example: `otel-logs-%{service.name}` or `otel-logs-%{custom.label}`
-  - The value is looked up from a context map (resource attributes, log attributes, etc.).
-  - If the key is missing, the value from `logs_index_fallback` is used (or `unknown` if not set).
-  - Only one placeholder is supported in the index name.
-  - The index names generated from these placeholders should adhere to [index naming restrictions](https://docs.opensearch.org/docs/latest/api-reference/index-apis/create-index/#index-naming-restrictions)
+- `logs_index` - Custom index name pattern for logs
+- `logs_index_fallback` - Fallback value when placeholder is missing (default: `unknown`)
+- `logs_index_time_format` - Time suffix format for logs
```

Comment:
It might be good to mention that it only supports a time suffix in UTC.

Reply:
Thanks Paul for the review: [307998a](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/41757/commits/307998ac0398471f83c614b947cbd3647049dc5b)